Makefile.gen.k8sapi.mk

```diff
@@ -0,0 +1,74 @@
+# DO NOT EDIT. Generated with:
+#
+#    devctl
+#
+#    https://github.com/giantswarm/devctl/blob/4c09e629d5ccd51a8d9247011315c2e35d4613d3/pkg/gen/input/makefile/internal/file/Makefile.gen.k8sapi.mk.template
+#
+
+# Directories.
+API_DIR := $(shell [ -d api ] &&  echo api || echo pkg/apis)# default to api, fall back to pkg/apis
+CRD_DIR := config/crd
+SCRIPTS_DIR := hack
+GOBIN_DIR := $(abspath hack/bin)
+CONTROLLER_GEN := $(abspath $(GOBIN_DIR)/controller-gen)
+
+# Colors
+BUILD_COLOR = ""
+GEN_COLOR = ""
+NO_COLOR = ""
+ifneq (, $(shell command -v tput))
+ifeq ($(shell test `tput colors` -ge 8 && echo "yes"), yes)
+BUILD_COLOR=$(shell echo -e "\033[0;34m")
+GEN_COLOR=$(shell echo -e "\033[0;32m")
+NO_COLOR=$(shell echo -e "\033[0m")
+endif
+endif
+
+# Inputs
+DEEPCOPY_BASE = zz_generated.deepcopy
+BOILERPLATE = $(SCRIPTS_DIR)/boilerplate.go.txt
+YEAR = $(shell date +'%Y')
+
+DEEPCOPY_FILES := $(shell find $(API_DIR) -name $(DEEPCOPY_BASE).go)
+
+all: generate
+
+$(CONTROLLER_GEN):
+	@echo "$(BUILD_COLOR)Building controller-gen$(NO_COLOR)"
+	GOBIN=$(GOBIN_DIR) go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.8.0
```

Comment:
```suggestion
	GOBIN=$(GOBIN_DIR) go install sigs.k8s.io/controller-tools/cmd/controller-gen@latest
```

Reply:
@QuantumEnigmaa hère :)

Reply:
Yeah but in my branch (and thus in the main one as well) it's already `latest`. See my above message :/

Reply:
Let's pair tomorrow :()

Reply:
Sure :)